src/test/kotlin/user/UserCommandTest.kt

```diff
@@ -0,0 +1,48 @@
+package cz.smarteon.loxone.user
+
+import cz.smarteon.loxone.LoxoneUuid
+import cz.smarteon.loxone.readResource
+import org.junit.jupiter.api.Test
+import strikt.api.expectThat
+import strikt.api.expectThrows
+import strikt.assertions.isEqualTo
+
+class UserCommandTest {
+
+    @Test
+    fun `should create commands`() {
+        val nfcTag = NfcTag(NFC_NAME, NFC_ID)
+        val userGroup = UserGroup(LoxoneUuid(GROUP_U))
+
+        expectThat(readResource<User>("user/user.json")) {
```

Comment:
pls slightly reword - we are not testing `readResource<User>`, we only use it as inut for tested commands, rather use
```kotlin
expect {
    that(UserCommand....).isEqualTo()
    that(UserCommand....).isEqualTo()
}
```